tests/run_tests/driver_dc_2d_test.py

```diff
@@ -129,26 +127,18 @@ def test_dc_2d_run(tmp_path: Path, max_iterations=1, pytest=True):
         output["data"] = potential.values[np.isfinite(potential.values)]
     if pytest:
         check_target(output, target_run)
-    else:
-        return driver.inverse_problem.model
 
 
 if __name__ == "__main__":
     # Full run
-    m_start = test_dc_2d_fwr_run(
+    test_dc_2d_fwr_run(
         Path("./"),
         n_electrodes=20,
         n_lines=3,
         refinement=(4, 8),
     )
-
-    m_rec = test_dc_2d_run(
+    test_dc_2d_run(
         Path("./"),
         max_iterations=20,
         pytest=False,
     )
-    residual = np.linalg.norm(m_rec - m_start) / np.linalg.norm(m_start) * 100.0
```

Comment:
How come?

Reply:
It is mostly just visual inspection that we are doing anyway on the full run. We've never really enforced those values (except for gravity), and kind of a very loose metric, so why bother?